README.md

```diff
@@ -23,16 +23,16 @@ For DevOps or SRE teams this charm will make operating Synapse simple and
 straightforward through Juju's clean interface. It will allow easy deployment
 into multiple environments for testing of changes.
 
+## Get started
+
+To begin, refer to the [Getting Started](https://charmhub.io/synapse/docs/tutorial-getting-started)
+tutorial for step-by-step instructions.
+
+## Learn more
+* [Read more](https://charmhub.io/synapse)
+
```

Comment:
Are there developer docs that we can link to here, perhaps https://element-hq.github.io/synapse/latest/index.html? 

(I honestly unsure about whether that's the correct or useful page for developer docs -- I'm keen to hear your thoughts)

Is there an official webpage for Synapse?

Reply:
Hmm. Actually, I thought that the links in this section would be related only to the charm, not the application that the charm provides. That's why I only added the "Read more."

Should I add links to Synapse for the following items then?
```
* [Developer documentation]() <!--Link to any developer documentation-->
* [Official webpage]() <!--(Optional) Link to official webpage/blog/marketing content--> 
* [Troubleshooting]() <!--(Optional) Link to a page or section about troubleshooting/FAQ-->
```

Reply:
Thanks for the clarification; that's good to know (and might warrant a change in the template repo to clarify for future READMEs)

Yes, I would say to add links for those three items. If there's no good page for Troubleshooting, then I say to add the link to the Matrix channel there instead. Thanks again!

Reply:
Thanks, now I added the correct links :)

Reply:
Great, thanks!